main.go

```diff
@@ -239,6 +239,13 @@ func main() {
 			os.Exit(1)
 		}
 	}
+
+	if cfg.Internal.NativeSidecarSupport {
```

Comment:
no need for a if statement  `setupLog.Info("Native sidecar", cfg.Internal.NativeSidecarSupport)`